aiographite/aiographitesend.py

```diff
@@ -0,0 +1,449 @@
+import asyncio
+from aiographite.graphite_escaping import metrics_name_to_graphite, metrics_name_from_graphite
+import pickle
+import struct
+import socket
+import os
+import time
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AsyncioGraphiteSendService(object):
```

Comment:
Also, we don't really need "SendService". 

- the word 'service' is unnecessary, it doesn't describe the class any better.
- the word 'send' is unnecessary. It's clear from context that this class can send:

```
graphite = AsyncIOGraphite()
graphite.send_single_data()
```